ansible/roles/bie-index/templates/bie-index-config.yml

```diff
@@ -40,7 +40,7 @@ security:
 {% endif %}
   jwt:
     discoveryUri: {{ discoveryUri | default('') }}
-    clientId: {{ (bie_index_clientId | default(clientId)) | default('') }}
+    clientId: {{ (bie_index_client_id | default(clientId)) | default('') }}
```

Comment:
This was a typo in a previous patch. We use `bie_index_client_id` and ALA `clientId`.